activitysim/examples/prototype_mtc_extended/configs/disaggregate_accessibility.yaml

```diff
@@ -0,0 +1,159 @@
+            bldgsz: 2                    # Building size - Single family detached
+        # Additional columns that are mapped to another (e.g., hhinccat1: 1 = hinc: 14000)
+        mapped_fields:
+            hinccat1: # List new fields mapped to this field
+              income: # Median income within each quartile
+                1: 15000   # 14000  (for 3 level)
+                2: 45000   # 67000  (for 3 level)
+                3: 80000   # 120000 (for 3 level)
+                4: 150000
+            persons:
+              hhsize:
+                1: 1
+                2: 2
+                3: 3
+            hworkers:
+              num_workers:
+                1: 1
+                2: 2
+                3: 3
+                    
+    PROTO_PERSONS:
+        index_col: proto_person_id
+        VARIABLES:
+            pnum: [1, 2]                # Person number
+            military: 4                 # Not military
+            pstudent: 3                 # Not attending school
+            educ: 13                    # Bachelor's
+            grade: 0                    # Not attending
+            timeFactorWork: 1           # mean
+            timeFactorNonWork: 1        # mean
+        mapped_fields:
+            pnum:
+                age:
+                    1: 35
+                    2: 55
+                sex: # Female male
+                    1: 2
+                    2: 1
+                ptype: # Person type, full-time and non-working adult
+                    1: 1
+                    2: 4
+                pemploy: # Full-time and unemployed
+                    1: 1
+                    2: 3
+                weeks: # 50-52 weeks, none
+                    1: 1
+                    2: 0
+                hours: # Hours per week
+                    1: 35
+                    2: 0
+                DAP: # Mandatory, Non-mandatory
+                    1: "M"
+                    2: "N"
+
+    PROTO_TOURS:
+        index_col: proto_tour_id
+        VARIABLES:
+            tour_num: 1                 # Tour number, 1 tour per person
+            purpose: [1, 2, 3]
+        mapped_fields:
+            purpose:
+                person_num: # In this case it was easier to map the person number directly to the purposez
+                    1: 1
+                    2: 2
+                    3: 2
+                tour_type:
+                    1: "work"
+                    2: "shopping"
+                    3: "othdiscr"
+                tour_category:  # tour purpose category, mandatory/non-mandatory
+                    1: "mandatory"
+                    2: "non_mandatory"
+                    3: "non_mandatory"
+        filter_rows: # list any matching conditions as pandas expression
+            - ~((df.tour_type == "work") & (df.person_num == 1))
+        JOIN_ON:
+            person_num: pnum            # Specifies which person variable to join the tours on
+
+# Merge on variables
```

Comment:
Based on the code, the user can also specify `asof` and `method`. It would be good to have some documentation on that without having user to look at the code, even though they might be less likely used.